src/screens/OrgPost/OrgPost.tsx

```diff
@@ -237,7 +237,7 @@ function orgPost(): JSX.Element {
                         {t('Text')}
                       </Dropdown.Item>
                       <Dropdown.Item
-                        value="searchTitle"
+                        // value="searchTitle"
```

Comment:
This PR is not related to OrgPost, so please revert any change you made to this file